src/OrchardCore.Modules/OrchardCore.Users/Controllers/ChangeEmailController.cs

```diff
@@ -49,7 +55,7 @@ public async Task<IActionResult> Index()
 
             var user = await _userService.GetAuthenticatedUserAsync(User);
 
-            return View(new ChangeEmailViewModel() { Email = ((User)user).Email });
+            return View(new ChangeEmailViewModel(_emailAddressValidator) { Email = ((User)user).Email });
```

Comment:
The service needs to be resolved in the Vallidate method

Reply:
OK: Resolve EmailAddressValidator from Validate method